src/IECoreImage/ImageReader.cpp

```diff
@@ -107,16 +107,44 @@ class ImageReader::Implementation
 				{
 					DeepData deepData;
 					ImageInput *input = ImageInput::open( m_inputFileName.c_str() );
-					// this will return false for deep tile image, we should probably find a way to
-					// support verifying integrity for deep tile image as well.
-					return input->read_native_deep_scanlines(
-						spec->height + spec->y - 1,
-						spec->height + spec->y,
-						0, // first deep sample
-						0, // first channel
-						spec->nchannels, // last channel
-						deepData
-					);
+
+					// Note that the spec we get from the image cache has a tiling setting
+					// based on the caching settings, not the file on disk, so we have to
+					// look at this disk spec.  Once we upgrade to modern OIIO, we should
+					// use "dimension_spec()" here, which gets us just the information we need.
```

Comment:
I'm ok with this for now, but its a little weird since we could just upgrade OIIO now (as far as public builds are concerned) but we're a bit locked to 1.8.X at IE for a while....